fuse_models/package.xml

```diff
@@ -13,6 +13,7 @@
 
   <build_depend>roslint</build_depend>
 
+  <depend>benchmark</depend>
```

Comment:
The `benchmark` rosdep entry does not exist in the standard ROS [repo](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml). If you remove it, I expect the buildfarm to build successfully, as it does not know about the `-DCATKIN_ENABLE_BENCHMARKING` flag. I'll consult with our build team, but I suspect benchmark results from our CI system would be largely meaningless anyway. And I can install libbenchmark and run the benchmarks when needed.

Reply:
True, we have it in a private `rosdep` set. You'd only need this:
``` bash
benchmark:
  ubuntu: benchmark
```
The debian is called `benchmark`. I actually created the debian we use, but I think it comes with versions of Ubuntu newer than Xenial. The debian name might change though.

Let me know if you want to give that a try. Otherwise, I can simply remove the dependency, and that should let the checks pass. You'll have to test things locally though, to confirm this builds and the benchmark runs.

Reply:
I believe this is the official ubuntu package: [libbenchmark-dev](https://packages.ubuntu.com/bionic/libbenchmark-dev)
At least for Ubuntu Bionic. I don't see a package for Xenial.

We do indeed have our own rosdep set, and I'll look at getting this added to it. But since this needs to be buildable without access to our private rosdep files, I'm going to ask that package dependency be removed.

I have successfully run the benchmarks locally, and see a similar large performance gain from the analytic derivatives. Thanks for this.

Reply:
I commented out the `depend` entry for `benchmark` and added a comment explaining why and what needs to happen to uncomment it. I think we could actually send a PR to add it to the rosdep.